pallets/order-book/src/lib.rs

```diff
@@ -1086,6 +1087,70 @@ impl<T: Config> LiquiditySource<T::DEXId, T::AccountId, T::AssetId, Balance, Dis
         }
     }
 
+    fn step_quote(
+        dex_id: &T::DEXId,
+        input_asset_id: &T::AssetId,
+        output_asset_id: &T::AssetId,
+        amount: QuoteAmount<Balance>,
+        _samples_count: usize,
```

Comment:
Should this parameter be even included in the interface, if one of the implementations ignores it? Sounds tricky to use with implicit/unexpected behaviour

Reply:
But we need to provide it to another sources. It is ok if some implementations don't use all attribites from time to time

Reply:
Hmm, maybe call it something like `samples_count_default` so that the intent is more clear when using the trait?

Reply:
`recommended_samples_count`